docker/quickstart.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Quickstarts DataHub by pullinng all images from dockerhub and then running the containers locally. No images are
+# built locally. Note: by default this pulls the latest version; you can change this to a specific version by setting
+# the DATAHUB_VERSION variable.
```

Comment:
*environment* variable, to avoid confusion

Reply:
Done